src/test/java/org/apache/tomcat/jakartaee/ManifestConverterTest.java

```diff
@@ -0,0 +1,16 @@
+package org.apache.tomcat.jakartaee;
+
+import static org.junit.Assert.assertTrue;
```

Comment:
Please do not use static import.

Reply:
I reviewed all the unit test code in the jakartaee package, and it appears that all of them use static imports for Assert.

Reply:
You're right, But the reason it's not recommended is that it's not very intuitive because we don't know if it's a junit method or a custom method when the test has many code.